### DISCUSSION TITLE
How do I remove emulators post installation?

### DISCUSSION THREAD

**Discussion by ElusiveZatchmo** [OnionUI/Onion#526] [Q&A]
How do I remove emulators I decided I no longer want once I've already installed and setup onion? I've been looking around and can't seem to find anything in this regard. The read me says that all actions are reversible, but forgive my confusion because I'm not sure how to accomplish that.

**Comment by Artickw** [comment]
Just open Onion Installer again and uncheck the boxes for the emulators you don't want anymore.

**Reply by RoGs7** [reply]
thats doesnt appear my friend......another way to uninstall some emulators we dont use anymore?

**Reply by Olywa** [reply]
It is there. Go to **Apps**-->**The Onion Installer**. 
If you do not have this then something is wrong with your installation as it is an integral part of Onion OS.

**Comment by Olywa** [comment]
@ElusiveZatchmo  As above, you can find 'The Onion Installer' in Apps
